src/Microsoft.IdentityModel.Tokens/SecurityTokenDescriptor.cs

```diff
@@ -71,6 +71,12 @@ public class SecurityTokenDescriptor
         /// </summary>
         public DateTime? NotBefore { get; set; }
 
+        /// <summary>
+        /// Gets or sets the token type
+        /// <remarks> This value will be replaced in the JWS header(in case of no encryption) and in outer header of JWE by 'typ' claim from <see cref="AdditionalHeaderClaims"/>, if it is provided.</remarks>
```

Comment:
Is the 'typ' value added to both headers if the token happens to be encrypted? If so, it might be worth adding that in since I think it's a little unclear.

"If provided, this will be added as the value for the 'typ' header parameter. In the case of a JWE, this will be added to both the inner (JWS) and the outer token (JWE) header. By default, the value used is 'JWT'.

"If AdditionalHeaderClaims also contains 'typ' header claim value, it will override the TokenType provided here."

Reply:
Good idea to mention default value as well. I made the change.